### PR TITLE
Added new objects for sds native alert

### PR DESCRIPTION
**Pull request by GowthamShanmugam**
This kind of alerts does not have clearing alert so
it is never comes under alert condtion. so we are
storing these aler in /alerting/notify.

tendrl-bug-id : Tendrl/commons#730
Signed-off-by: GowthamShanmugam <gshanmug@redhat.com>

### CODE REVIEW

**Comment by shtripat** [comment]
Where is the logic for persisting these alerts? Also I think rather we should call this as `UtilizationAlert` which is meant for saving alerts for which there are not going to be clearing events.

**Comment by GowthamShanmugam** [comment]
@shtripat modified

**Comment by GowthamShanmugam** [comment]
@shtripat logic is in https://github.com/Tendrl/node-agent/pull/618

File: tendrl/commons/objects/definition/master.yaml [hunk 1]
```diff
@@ -759,6 +759,63 @@ namespace.tendrl:
       value: alerting/clusters/$Alert.tags['integration_id']/$Alert.alert_id
       list: alerting/clusters/$Alert.tags['integration_id']
       help: "Cluster alerts"
+    UtilizationOnlyAlert:
```
Comment (r0h4n) [inline tendrl/commons/objects/definition/master.yaml]: What is the connection with "Utilization" in the name? @shtripat @GowthamShanmugam

Comment (GowthamShanmugam) [reply tendrl/commons/objects/definition/master.yaml]: @shtripat  i am also felt this name will confuse.. need to change this object name

Comment (shtripat) [reply tendrl/commons/objects/definition/master.yaml]: Ah my bad I meant `NotificationOnlyAlert` and not `UtilizationOnlyAlert` :(
This alert class represents alerts for which no clearing events would come later.

**Pull request merged**